sentry/src/main/java/io/sentry/protocol/Device.java

```diff
@@ -93,7 +93,18 @@
   private @Nullable TimeZone timezone;
 
   private @Nullable String id;
-  private @Nullable String language;
+
+  /**
+   * This method returns the language code for this locale, which will either be the empty string or
```

Comment:
`m`: I think it's worth mentioning that `language` is the same as `locale`, but it was a mistake and we can't change it, because it would be a breaking change.

Reply:
it returns the language only now, see the PR description